index.html

```diff
@@ -289,6 +289,80 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <form class="contact-form">
```

Comment:
as pointed out by the previous reviewer, It is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) that you should be able to collect data from your form with the use of the [Formspre service](https://formspree.io/html/).
currently, Your form lacks that functionality. kindly fix it😉😉
<img width="257" alt="formsprre" src="https://user-images.githubusercontent.com/64643995/211061826-8270a9ee-b376-46aa-9663-0c5124bd5aae.png">